optionals/compat_rhs_usf3/CfgWeapons.hpp

```diff
@@ -14,6 +15,98 @@ class CfgWeapons
     class rhs_weap_m4_Base: arifle_MX_Base_F {
         ACE_barrelTwist=177.8;
         ACE_barrelLength=368.3;
+		class M203_GL : UGL_F {
+			magazines[] = {
+				"rhs_mag_M441_HE",
+				"rhs_mag_M433_HEDP",
+				"rhs_mag_M4009",
+				"rhs_mag_m576",
+				"rhs_mag_M585_white",
+				"rhs_mag_M661_green",
+				"rhs_mag_M662_red",
+				"rhs_mag_M713_red",
+				"rhs_mag_M714_white",
+				"rhs_mag_M715_green",
+				"rhs_mag_M716_yellow",
+
+				//bis compatibility
+				"1Rnd_HE_Grenade_shell",
+				"UGL_FlareWhite_F",
+				"UGL_FlareGreen_F",
+				"UGL_FlareRed_F",
+				"UGL_FlareYellow_F",
+				"UGL_FlareCIR_F",
+				"1Rnd_Smoke_Grenade_shell",
+				"1Rnd_SmokeRed_Grenade_shell",
+				"1Rnd_SmokeGreen_Grenade_shell",
+				"1Rnd_SmokeYellow_Grenade_shell",
+				"1Rnd_SmokePurple_Grenade_shell",
+				"1Rnd_SmokeBlue_Grenade_shell",
+				"1Rnd_SmokeOrange_Grenade_shell",
+				"3Rnd_HE_Grenade_shell",
+				"3Rnd_UGL_FlareWhite_F",
+				"3Rnd_UGL_FlareGreen_F",
+				"3Rnd_UGL_FlareRed_F",
+				"3Rnd_UGL_FlareYellow_F",
+				"3Rnd_UGL_FlareCIR_F",
+				"3Rnd_Smoke_Grenade_shell",
+				"3Rnd_SmokeRed_Grenade_shell",
+				"3Rnd_SmokeGreen_Grenade_shell",
+				"3Rnd_SmokeYellow_Grenade_shell",
+				"3Rnd_SmokePurple_Grenade_shell",
+				"3Rnd_SmokeBlue_Grenade_shell",
+				"3Rnd_SmokeOrange_Grenade_shell",
+				
+				//ACE3 Compatibility
+				"ACE_HuntIR_M203"
+			};
+		};
+		class M320_GL : M203_GL {
+			magazines[] = {
+				"rhs_mag_M441_HE",
+				"rhs_mag_M433_HEDP",
+				"rhs_mag_M4009",
+				"rhs_mag_m576",
+				"rhs_mag_M585_white",
+				"rhs_mag_M661_green",
+				"rhs_mag_M662_red",
+				"rhs_mag_M713_red",
+				"rhs_mag_M714_white",
+				"rhs_mag_M715_green",
+				"rhs_mag_M716_yellow",
+
+				//bis compatibility
+				"1Rnd_HE_Grenade_shell",
+				"UGL_FlareWhite_F",
+				"UGL_FlareGreen_F",
+				"UGL_FlareRed_F",
+				"UGL_FlareYellow_F",
+				"UGL_FlareCIR_F",
+				"1Rnd_Smoke_Grenade_shell",
+				"1Rnd_SmokeRed_Grenade_shell",
+				"1Rnd_SmokeGreen_Grenade_shell",
+				"1Rnd_SmokeYellow_Grenade_shell",
+				"1Rnd_SmokePurple_Grenade_shell",
+				"1Rnd_SmokeBlue_Grenade_shell",
+				"1Rnd_SmokeOrange_Grenade_shell",
+				"3Rnd_HE_Grenade_shell",
+				"3Rnd_UGL_FlareWhite_F",
+				"3Rnd_UGL_FlareGreen_F",
+				"3Rnd_UGL_FlareRed_F",
+				"3Rnd_UGL_FlareYellow_F",
+				"3Rnd_UGL_FlareCIR_F",
+				"3Rnd_Smoke_Grenade_shell",
+				"3Rnd_SmokeRed_Grenade_shell",
+				"3Rnd_SmokeGreen_Grenade_shell",
+				"3Rnd_SmokeYellow_Grenade_shell",
+				"3Rnd_SmokePurple_Grenade_shell",
+				"3Rnd_SmokeBlue_Grenade_shell",
+				"3Rnd_SmokeOrange_Grenade_shell",
+				
+				//ACE3 Compatibility
+				"ACE_HuntIR_M203"
+			};
+		};
     };
```

Comment:
Sorry, I'm very new at, well everything, so I need a bit of an explanation. Should I replace tabs with four spaces?
If so (so I know how to do stuff in the future), is that the project standard or a github standard or something? I just use tabs for visibility purposes :)